README.md

```diff
@@ -7,7 +7,7 @@
 [![codecov](https://codecov.io/github/eth-easl/modyn/graph/badge.svg?token=KFDCE03SQ4)](https://codecov.io/github/eth-easl/modyn)
 [![License](https://img.shields.io/github/license/eth-easl/modyn)](https://img.shields.io/github/license/eth-easl/modyn)
 
```

Comment:
We should get https://github.com/eth-easl/modyn/pull/442 merged at some point ;) I'll take a look at it this week as I suspect that some details need to be adjusted after having completed the implementation now :)